addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -35,6 +39,21 @@
     flex: 0 0 auto;
     height: 41px; // match minimal-height of input, including border width
 
+    &:not(.o-composer-is-compact) {
+        border: 0;
+        height: auto;
+        padding: 0 10px;
+        width: 100%;
+
+        .o_Composer_toolButtons{
+            flex-direction: row;
+            justify-content: space-between;
+            flex: 100%;
+            border-bottom: 0;
+            border-radius: initial;
+        }
```

Comment:
put a modifier class on this component instead (avoid CSS specificity when possible, ideal exception is modifier on self)